configuration/sitemaps.md

```diff
@@ -471,33 +481,46 @@ mappings=[15="Gone", 19="Chilly", 21="Cozy"]
 
 As you can see, different Item data types are accepted as mappings values.
 The first two lines show very typical use cases.
-Imagine your TV as part of your openHAB setup.
-It's power state and channel number are internally represented by a binary Switch Item (OFF/ON) and a discrete number Item with only a few selectable states.
-By using a Switch or Selection element with a mappings array, you can replace these meaningless values by meaningful descriptions for your inputs in the user interface.
+Imagine your TV is part of your openHAB setup.
+Its power state is represented by a binary Switch Item.
+Its channel number is a discrete number Item that may only be set to one of three states.
+By using a Switch or Selection element with a mappings array, you can replace these meaningless values with user-friendly descriptions for display on the user interface.
+Using this mapping, the power state of the TV, instead of saying "ON" and "OFF" would more accurately display "on" and "standby".
+Similarly, mapping above changes the numbers "1", "2", and "3" to "DasErste", "BBC One", and "Cartoon Network" respectively.
 
-In the third and forth line only a subset of the possible values of items belonging to a heating system is presented to the user.
-This limits the possible input values, which is yet another often occurring use case.
+In the third and forth examples above, only a subset of the possible values of items belonging to a heating system are presented to the user.
+This limits the possible input values, which is yet another often occurring use case for mappings.
 
 ## Dynamic Sitemaps
 
-All Sitemap elements can be configured to be hidden, color highlighted or to have a dynamic icon, depending on certain Item states.
+All Sitemap elements can be configured to be hidden, color highlighted or to have a [dynamic icon]({{base}}/items.md#icons-dynamic), depending on certain Item states.
 A few practical use cases are:
 
-- Show a battery warning if the voltage level of a device is below 30%.
-- Hide further control elements for the TV if it is turned off.
-- Highlight a value with a warning color if it is outside accepted limits.
-- Present a special icon, depending on the state of an item.
+- Show a battery warning if the voltage level of a device is below 30%
+- Hide further control elements for the TV if it is turned off
+- Highlight a value with a warning color if it is outside accepted limits
+- Present a special icon, depending on the state of an item (a [dynamic icon]({{base/items.md#icons-dynamic}}))
 
 ### Visibility
 
-To dynamically show or hide an item, the `visibility` parameter is used.
-By default, an Item is visible if the `visibility` parameter is not provided.
+The `visibility` parameter is used to dynamically show or hide an Item.
+If the parameter is not provided, the default is to display the Item.
+
+Visibility syntax:
```

Comment:
See above. I believe these are superfluous?

Reply:
Hum - not sure how this is superflurious - or even not needed ;-)

Reply:
Basically I don't like these two-word paragraphs. They tend to look not good in the end result. However yes sure, without them the read flow is interrupted by a sudden code block. Do as you see fit

Reply:
This comment has been addressed.